cmd/limactl/edit.go

```diff
@@ -100,7 +101,7 @@ func editAction(cmd *cobra.Command, args []string) error {
 		hdr += "# and an empty file will abort the edit.\n"
 		hdr += "\n"
 		hdr += editutil.GenerateEditorWarningHeader()
-		yBytes, err = editutil.OpenEditor(yContent, hdr)
+		yBytes, err = editutil.OpenEditor(context.TODO(), yContent, hdr)
```

Comment:
```suggestion
		yBytes, err = editutil.OpenEditor(ctx, yContent, hdr)
```

w/ `ctx := cmd.Context()`